src/layout/SideBarLayout.tsx

```diff
@@ -4,13 +4,15 @@ import BaseLayout from './BaseLayout';
 import './styles/SideBarLayout.scss';
 
 type SideBarLayoutProps = {
+  title: JSX.Element;
   sidebar: JSX.Element;
   content: JSX.Element;
 };
 
-const SideBarLayout: React.FC<SideBarLayoutProps> = ({ sidebar, content }) => (
+const SideBarLayout: React.FC<SideBarLayoutProps> = ({ title, sidebar, content }) => (
   <BaseLayout>
     <section id="sidebar-layout" data-layout="left-sidebar-layout">
```

Comment:
The BEM guideline explicitly bans the usage of any IDs or Tag names for the CSS identifiers e.g. `id="sidebar-layout"` should be changed to `className="sidebar-layout"` and the related CSS code should change too.

Reply:
Woops!